x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/schema.ts

```diff
@@ -167,4 +168,28 @@ export const cspmUsageSchema: MakeSchemaFrom<CspmUsage> = {
       alerts_acknowledged_count: { type: 'long' },
     },
   },
+  cloud_account_stats: {
+    type: 'array',
+    items: {
+      account_id: { type: 'keyword' },
+      cloud_provider: { type: 'keyword' },
+      product: { type: 'keyword' },
+      package_policy_id: { type: 'keyword' },
+      latest_doc_count: { type: 'long' },
+      latest_doc_updated_timestamp: { type: 'date' },
+      posture_management_stats: {
+        posture_score: { type: 'long' },
+        benchmark_name: { type: 'keyword' },
+        benchmark_version: { type: 'keyword' },
+        passed_findings_count: { type: 'long' },
+        failed_findings_count: { type: 'long' },
+      },
+      kspm_stats: {
+        kubernetes_version: { type: 'keyword' },
+        agents_count: { type: 'short' },
```

Comment:
It can be but I believe we account for `agents_count` for the number of agents being deployed on cluster nodes. See [Resources Dashboard](https://lookerstudio.google.com/u/0/reporting/a2882d8b-6051-40e6-af1f-c120aa4cbe52/page/p_re9yivfk5c?s=qorpDqoUROQ)

Reply:
I think that if this piece of data is relevant for all our products, it should be collected and placed in a higher level in the hierarchy

Reply:
Why do we care about cspm `agents_count` from a product metric perspective?